app/main.py

```diff
@@ -1 +1,43 @@
-#  write your code here
+from typing import List
+
+
+class Animal:
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True
+                 ) -> None:
```

Comment:
think is bad code style. must be:
```suggestion
                def __init__(
                self, name: str,
                 appetite: int,
                 is_hungry: bool = True
                 ) -> None:
```

Reply:
good point